### DISCUSSION TITLE
3 Containers

### DISCUSSION THREAD

**Discussion by hockeygoalie35** [djmaze/resticker#199] [Q&A] [answered]
Hi!
 Just curious, why is this split up into 3 separate containers? Why not have it all under one container, still 3 different cron schedules?

Thanks!

**Comment by djmaze** [comment, marked as answer]
I Iike the single responsibility principle and the one-process-per-container pattern. Also _go-cron_ as currently implemented only supports one schedule.

**Comment by hockeygoalie35** [comment]
Gotcha! Thanks for the quick answer, love the ease of your project.
